onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -541,6 +541,23 @@ TEST(GraphTransformationTests, MatMulAddFusion_three_input) {
   ASSERT_TRUE(op_to_count["Gemm"] == 1);
 }
 
+TEST(GraphTransformationTests, MatMulAddFusion_negitive_case) {
```

Comment:
>MatMulAddFusion_negitive_case [](start = 31, length = 29)

Can you make the name more explicit or add a comment stating what this is exactly testing, more specifically which negative case this test is testing.